.eslintrc.cjs

```diff
@@ -18,17 +18,26 @@ module.exports = {
     'es6': true,
     'node': true
   },
+  'plugins': [
+    '@typescript-eslint'
+  ],
   'extends': [
     'eslint:recommended',
     'google'
   ],
+  "parser": "@typescript-eslint/parser",
```

Comment:
nit: here and below, please use single quotes around property names and for string literals, for consistency with the rest of the file